### DISCUSSION TITLE
Maintainer for Rebass

### DISCUSSION THREAD

**Discussion by gamb** [system-ui/theme-ui#1700] [General]
Hey, I'm raising this issue in the hope that it reaches someone who might know how best to proceed with this semi-related project: https://github.com/rebassjs/rebass/issues/1083#issuecomment-828755335 

Rebass is part of the prior art of system-ui (as I understand it). There are currently a number of legitimate PRs to keep the project up to date with various dependencies, but nobody answering the threads is able to merge them.

It looks like it may be necessary to fork Rebass to keep it useable for those of us depending on it. I'm wondering it if could fit under the system-ui umbrella, or otherwise if there are suggestions from the community on who could help keep it alive.

Thanks,

Adam

**Comment by lachlanjc** [comment]
My understanding from Brent is that Theme UI is the successor to Rebass in his mind—I haven't used Rebass ever since Theme UI brought in components with near-zero migration. Are there features of Rebass that Theme UI is lacking? Personally, I'm already spending a bunch of hours per week on Theme UI so I'm not going to take on Rebass as well, but if there's features or migration docs that would help folks move over from Rebass, I'm happy to help.

**Reply by gamb** [reply]
Thanks @lachlanjc Migration to Theme UI may be the way forward. I'll take some time to understand the migration path myself. Then perhaps that's something we're able to feedback to existing Rebass users :)
